mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -2318,8 +2319,8 @@ class LogisticRegressionSuite
       assert(m1.interceptVector ~== m2.interceptVector absTol 0.05)
     }
     val testParams = Seq(
-      ("binomial", smallBinaryDataset, 2),
-      ("multinomial", smallMultinomialDataset, 3)
+      ("Binomial", smallBinaryDataset, 2),
```

Comment:
The changes you made don't address this comment at all, and there are not tests for the suggestion from Yanbo either.